Arc/Arc.js

```diff
@@ -170,6 +170,7 @@ const ArcDecorator = compose(
  * Usage:
  * ```
  * <Arc color="blue" endAngle={200} startAngle={0}" radius={100} />
+ * ```
```

Comment:
How can I see this on document? I think it's private. right?

Reply:
right. It was public before. So, @example and Usage is written